manual/oracle-sru-19.2.21.txt

```diff
@@ -0,0 +1,1177 @@
+# To disable cloud-init's network configuration capabilities, write a file
+# /etc/cloud/cloud.cfg.d/99-disable-network-config.cfg with the following:
+# network: {config: disabled}
+auto lo
+iface lo inet loopback
+
+# control-manual ens3
+iface ens3 inet dhcp
+    broadcast 10.0.0.255
+    dns-nameservers 169.254.169.254
+    gateway 10.0.0.1
+setup_proposed.sh                                                                                                                                                                100%  194     2.4KB/s   00:00    
+  cloud-init
+Get:1 http://archive.ubuntu.com/ubuntu xenial-proposed/main amd64 cloud-init all 19.2-24-ge7881d5c-0ubuntu1~16.04.1 [406 kB]
+Preparing to unpack .../cloud-init_19.2-24-ge7881d5c-0ubuntu1~16.04.1_all.deb ...
+Unpacking cloud-init (19.2-24-ge7881d5c-0ubuntu1~16.04.1) over (19.1-1-gbaa47854-0ubuntu1~16.04.1) ...
+Setting up cloud-init (19.2-24-ge7881d5c-0ubuntu1~16.04.1) ...
+Leaving 'diversion of /etc/init/ureadahead.conf to /etc/init/ureadahead.conf.disabled by cloud-init'
+cloud-init	19.2-24-ge7881d5c-0ubuntu1~16.04.1
+sudo: unable to resolve host SRU-didnt-work
+sudo: unable to resolve host SRU-didnt-work
+Connection to 129.146.132.127 closed by remote host.
+mkstemp: No such file or directory
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+ssh: connect to host 129.146.132.127 port 22: Connection refused
+.........
+status: done
+time: Wed, 04 Sep 2019 14:55:15 +0000
+detail:
+DataSourceOpenStack [net,ver=2]
```

Comment:
Post clean reboot, we've now detected DataSourceOpenStack instad of DataSourceOpenStackLocal so we are spending more time waiting until init-network boot stage to detect the datasource. I'm not entirely sure why that is on Oracle (and it may be behavior that existing before this upgrade).   Can we try a cloud-init clean --reboot --logs without an upgrade to -proposed to confirm that 19.1.1 has this behavior too.  I think this could be a bug that already exists (not a regression).